components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/util/EventHookHandlerUtils.java

```diff
@@ -0,0 +1,265 @@
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static com.wso2.identity.asgardeo.event.configuration.mgt.core.service.util.EventConfigurationMgtConstants.EVENT_CONFIG_SCHEMA_NAME_KEY;
+import static com.wso2.identity.asgardeo.event.configuration.mgt.core.service.util.EventConfigurationMgtConstants.ErrorMessages.ERROR_WHILE_CASTING_EVENT_CONFIG_AT_SERVER;
+import static org.wso2.carbon.identity.application.authentication.framework.util.FrameworkUtils.CORRELATION_ID_MDC;
+
+/**
+ * This class contains the utility method implementations.
+ */
+public class EventHookHandlerUtils {
+
+    private static final Log log = LogFactory.getLog(EventHookHandlerUtils.class);
+
+    /**
+     * Get the identity property specified in identity-event.properties
+     *
+     * @param moduleName   The name of the module which the property belongs to
+     * @param propertyName The name of the property which should be fetched
+     * @return The required property
+     */
+    public static String getIdentityEventProperty(String moduleName, String propertyName) {
+
+        // Retrieving properties set in identity event properties
+        String propertyValue = null;
+        try {
+            ModuleConfiguration moduleConfiguration = IdentityEventConfigBuilder.getInstance()
+                    .getModuleConfigurations(moduleName);
+
+            if (moduleConfiguration != null) {
+                propertyValue = moduleConfiguration.getModuleProperties().getProperty(propertyName);
+            }
+        } catch (IdentityEventException e) {
+            log.error("An error occurred while retrieving module properties because " + e.getMessage(), e);
+        }
+        return propertyValue;
+    }
+
+    /**
+     * Retrieve event uri.
+     *
+     * @param eventKey  Event key.
+     * @return Event uri string.
+     * @throws IdentityEventServerException If an error occurs.
+     */
+    public static String getEventUri(String eventKey) throws IdentityEventServerException {
+
+        try {
+            ResourceConfig eventConfigObject = EventConfigurationMgtUtils.getEventConfig(eventKey);
+            if (eventConfigObject.getConfigs() != null &&
+                    eventConfigObject.getConfigs().containsKey(EVENT_CONFIG_SCHEMA_NAME_KEY)) {
+                return (String) eventConfigObject.getConfigs().get(EVENT_CONFIG_SCHEMA_NAME_KEY);
+            } else {
+                throw new IdentityEventServerException("Event schema not found in the resource event config " +
+                        "for the eventKey: " + eventKey);
+            }
+        } catch (EventConfigurationMgtServerException e) {
+            throw new IdentityEventServerException(e.getErrorCode(), e.getMessage());
+        } catch (ClassCastException e) {
+            throw new IdentityEventServerException(ERROR_WHILE_CASTING_EVENT_CONFIG_AT_SERVER.getCode(),
+                    ERROR_WHILE_CASTING_EVENT_CONFIG_AT_SERVER.getMessage());
+        }
```

Comment:
Handle these two in a consistent way.

Reply:
Addressed with https://github.com/wso2-extensions/identity-webhook-event-handlers/pull/1/commits/6e511b91f52c4b71ccff73c989894c76b285ebf0